### PR TITLE
Feature/profile nudge

### PR DESCRIPTION
**Pull request by jmgasper**
Tweak for checking gigAvailability - we just want to make sure the value has been set, we don’t care if it’s true or false for profile completeness calculations

### CODE REVIEW

File: src/services/MemberService.js [hunk 1]
```diff
@@ -194,7 +194,7 @@ async function getProfileCompleteness (currentUser, handle, query) {
       data.bio = true
     }
     // TODO: Do we use the short bio or the "description" field of the member object?
-    if(item.traitId=="basic_info" && item.traits.data[0].gigAvailability) {
```
Comment (kkartunov) [inline src/services/MemberService.js:197]: Please move to personalization traits. There is the location where profile app will store that preference.

File: src/services/MemberTraitService.js [hunk 2]
```diff
@@ -206,13 +206,6 @@ async function updateTraits (currentUser, handle, data) {
     }
   })
 
-  if(await lookerService.isMemberVerified(member.userId)){
```
Comment (kkartunov) [inline src/services/MemberTraitService.js:209]: Why do we need to remove this?

**Review by kkartunov — CHANGES_REQUESTED** [review]

File: src/services/MemberTraitService.js (hunk 2 above)
Comment (jmgasper) [reply src/services/MemberTraitService.js:209]: We don't need to do this in the TraitService - only the member service.  This was just leftover test code that I failed to remove that's causing a problem in dev

File: src/services/MemberService.js (hunk 1 above)
Comment (jmgasper) [reply src/services/MemberService.js:197]: Updated, please check.

**Review by kkartunov — APPROVED** [review]

**Pull request merged**